src/hu_stuff.c

```diff
@@ -983,8 +983,41 @@ static void HU_widget_build_keys (void)
   // build text string whose characters call out graphic keys
   for (k = 0; k < 6; k++)
   {
+    int card;
+    const int keyblinkkeys = plr->keyblinkkeys[k%3];
+  
+    if (!(card = plr->cards[k]) && keyblinkkeys &&
+        plr->keyblinktics & KEYBLINKMASK)
+    {
+      switch (keyblinkkeys)
+      {
+        case KEYBLINK_EITHER:
```

Comment:
I'm really not sure I like the alternating card and skull key.

Reply:
How about the `keyorskull[]` array from Crispy which makes it display the key that*s actually available on the map?

Reply:
I'd prefer that card+skull Boom HUD icon.

Reply:
The combined keys icon is already used for doors that require both keys, whereas the alternating effect is for doors that require either of them; what you see on the HUD is exactly what you need to use the door.

Knowing that, do you still insist on replacing the alternating effect?

Reply:
> Knowing that, do you still insist on replacing the alternating effect?

No, it's alright, thank you!

Reply:
Alright.

Reply:
> How about the `keyorskull[]` array from Crispy which makes it display the key that*s actually available on the map?

Done in f2ea30ecd112f931da34a03e201046ff6f838b30.